easybuild/easyconfigs/s/supermagic/supermagic-20170824-foss-2017a.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'supermagic'
+version = '20170824'
+git_commit_id = '07311ad853'
+
+homepage = 'https://github.com/hpc/supermagic'
+description = """Very simple MPI sanity code. Nothing more, nothing less."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+sources = [{
+    'filename': 'supermagic-%(version)s.tar.gz',
+    'download_filename': '%s.tar.gz' % git_commit_id,
+    'extract_cmd': "tar xfvz %s",
```

Comment:
@jordiblasco No need to specify a custom `extract_cmd` here, the one that is derived by default based on the `.tar.gz` extension works fine (`extract_cmd` should only be specified when really needed to unpack correctly).